### ISSUE TITLE
GitRepository.getOriginUrl is deprecated.

### ISSUE THREAD

**Issue by ivajo26** [jacogr/atom-git-control#81]
Use ::getOriginURL instead.
```
GitRepository.getOriginUrl (/usr/share/atom/resources/app.asar/src/git-repository.js:446:7)
Object.hasOrigin (/home/ivajo26/.atom/packages/git-control/lib/git.coffee:127:16)
GitControlView.fetchMenuClick (/home/ivajo26/.atom/packages/git-control/lib/git-control-view.coffee:161:24)
GitControlView.update (/home/ivajo26/.atom/packages/git-control/lib/git-control-view.coffee:68:9)
Object.updateViews (/home/ivajo26/.atom/packages/git-control/lib/git-control.coffee:44:8)
<unknown> (/home/ivajo26/.atom/packages/git-control/lib/git-control.coffee:17:56)
```

**Comment by nounoursheureux** [comment]
It should have been fixed by 008fe74 , please update the package and tell us if the issue persists :+1:

**Comment by MarcelMue** [comment]
Closed for now, feel free to reopen if the deprecation still exists somehow.
